### DISCUSSION TITLE
not able to change the confirmation email template in Self hosted supabase

### DISCUSSION THREAD

**Discussion by atularvind** [supabase/supabase#22347] [Questions]
I was trying to change the confirmation email template of the self-hosted supabase on my local machine but was not able to.

I created an email template and made it available at `http://localhost:3000/templates/email/confirm.html` and added the below config on the .env file
`MAILER_TEMPLATES_CONFIRMATION="http://localhost:3000/templates/email/confirm.html"`

and also added to the `docker-compose.yml` file as below 

`GOTRUE_MAILER_TEMPLATES_CONFIRMATION: ${MAILER_TEMPLATES_CONFIRMATION}`

and restarted the docker, but when I did the signup, it still uses the default confirmation email template, 

can anybody tell me what I am missing here?

**Comment by charislam** [comment]
Hey @atularvind, can you try if this works for you? https://supabase.com/docs/guides/cli/customizing-email-templates
